parser-typechecker/unison-parser-typechecker.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.4.
```

Comment:
Hmm, your `stack` has a newer `hpack`... Any advice @mitchellwrosen ?

Reply:
I'm not sure... maybe we should all just install stack via the same nixpkgs snapshot.